variables.tf

```
@@ -242,3 +242,9 @@ variable "user_group_ids" {
   default     = []
   description = "User Group ID to associate with the replication group"
 }
+
+variable "auto_minor_version_upgrade" {
+  type        = bool
+  default     = true
```

Should this default to false ?
```suggestion
  default     = false
```